boards/megafly_rev4/megafly_rev4.hpp

```diff
@@ -157,7 +159,8 @@ class Megafly_rev4
     File_flash_avr32            file_flash;
```

Comment:
general remarks:
Aren't we aiming at having doxygen documenting all those members ? :)